### PR TITLE
Change configuration for tests in CI to use oauth2

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Fix for https://github.com/owncloud/web/issues/4741.
This PR changes the ci configuration to run the tests with oauth2 login as opposed to openid login that was being used here. The CI build started failing since friday last week. That may be because of some error on openid connect setup. After discussion with @LukasHirt last week, we decided to change the setup back to oauth2 since that seems to be more common setup for oc10 + web. All the necessary setup  for openidconnect login is still here and we can easily enable it after a proper support for oc10 + openidconnect + web comes.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/4741.

## How Has This Been Tested?

:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: nightwatch.conf.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const LDAP_BASE_DN = process.env.LDAP_BASE_DN || 'cn=admin,dc=owncloud,dc=com'
 const LDAP_ADMIN_PASSWORD = process.env.LDAP_ADMIN_PASSWORD || 'admin'
 const OCIS_SKELETON_DIR = process.env.OCIS_SKELETON_DIR || './tests/testing-app/data/webUISkeleton/'
 const OPENID_LOGIN = RUN_ON_OCIS || !!process.env.OPENID_LOGIN
-const WEB_UI_CONFIG = process.env.WEB_UI_CONFIG || path.join(__dirname, 'dist/config.json')
+const WEB_UI_CONFIG = process.env.WEB_UI_CONFIG || path.join(__dirname, 'config/config.json')
```
Comment (individual-it) [inline nightwatch.conf.js:34]: why do we need this change? in https://github.com/owncloud/web/pull/4740/files#diff-6a968d4b7901c8526c8d9a86d1f434737dbfe2b00da21a7d3041ed4e3d58d62eR1055 you copy the config file to `dist`

File: tests/acceptance/expected-failures-with-oc10-server copy-openid-login.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIPreview/imageMediaViewer.feature:154](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L154)
 
 ### [WebUI Login](https://github.com/owncloud/web/issues/4677)
```
Comment (individual-it) [inline tests/acceptance/expected-failures-with-oc10-server copy-openid-login.md:24]: is that filename intended? `tests/acceptance/expected-failures-with-oc10-server copy-openid-login.md`

File: tests/acceptance/features/webUILogin/adminBlocksUser.feature [hunk 3]
```diff
@@ -14,3 +15,14 @@ Feature: view profile
     Then the user should be redirected to the login error page
     When the user exits the login error page
     Then the user should be redirected to the login page
+
+  @oauthLogin @notToImplementOnOCIS
```
Comment (individual-it) [inline tests/acceptance/features/webUILogin/adminBlocksUser.feature:19]: why shouldn't that be implemented on OCIS?

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/stepDefinitions/loginContext.js [hunk 4]
```diff
@@ -102,6 +107,10 @@ Then('the user should be redirected to the login page', function() {
   return client.page.loginPage().waitForPage()
 })
 
+Then('the user should be redirected to user disabled page', function() {
```
Comment (phil-davis) [inline tests/acceptance/stepDefinitions/loginContext.js:110]:
```suggestion
Then('the user should be redirected to the user disabled page', function() {
```

File: tests/acceptance/features/webUILogin/adminBlocksUser.feature (hunk 3 above)
Comment (dpakach) [reply tests/acceptance/features/webUILogin/adminBlocksUser.feature:19]: ocis doesn't support oauth2 login, it comes with openid by default

File: tests/acceptance/expected-failures-with-oc10-server copy-openid-login.md (hunk 2 above)
Comment (dpakach) [reply tests/acceptance/expected-failures-with-oc10-server copy-openid-login.md:24]: Mistake while copy pasting, i'll fix that

File: nightwatch.conf.js (hunk 1 above)
Comment (dpakach) [reply nightwatch.conf.js:34]: you're right, that's not required, I'll change that

**Review by phil-davis — APPROVED** [review]
LGTM - let's see what CI  thinks.
Don't worry about adding "the" to the new step text - we can review all the existing steps for this sort of grammar later.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13179/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13182/

<p>





</p></details>

**Review by phil-davis — APPROVED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**